flink-streaming-java/src/test/java/org/apache/flink/streaming/api/SourceFunctionTest.java

```diff
@@ -44,6 +45,18 @@ public void fromElementsTest() throws Exception {
 	}
 
 	@Test
+	public void fromElementsWithBaseTypeTest1() {
```

Comment:
these tests appear to be in the wrong file;I would put them into the StreamExecutionEnvironment class.